thoth/slo_reporter/sli_apis/sli_user_api.py

```diff
@@ -137,6 +142,19 @@ def _report_sli(self, sli: Dict[str, Any]) -> str:
         @param sli: It's a dict of SLI associated with the SLI type.
         """
         html_inputs = self._evaluate_sli(sli=sli)
+        sli_path = f"{self._SLI_NAME}/{self._SLI_NAME}-{self.configuration.last_week_time}.csv"
```

Comment:
I think this can go in the base class, because it is the same for all classes, WDYT? as we have `default_columns` for example.

Reply:
I think we agreed to leave as is for now.